source/includes/connection-snippets/scram-snippets/csharp-connection.cs

```diff
@@ -0,0 +1,9 @@
+// begin scram connection
+using MongoDB.Bson;
+using MongoDB.Driver;
+// ...
+var client = new MongoClient(
+    "mongodb+srv://<username>:<password>@<cluster-address>/test?w=majority"
```

Comment:
Nitpick: sometimes the connection string includes `cluster-url` and others it includes `cluster-address`. Is this on purpose and if so, what is the reason?

Reply:
Fixed. All switched to `<cluster-url>`